### PR TITLE
Add commits page styling

### PR DESCRIPTION
**Pull request by TimvdLippe**
This adds the styling for the data that @mpasterkamp retrieves.

### CODE REVIEW

File: src/tindev-commits.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        margin: 0 10px 10px 0;
+      }
+
+      .code {
+        clear: left;
+      }
+    </style>
+    <iron-ajax
+      auto
+      url="https://api.github.com/repos/polymer/polymer/commits/3d054bab6645421878598d26498a9f2a47804b5d"
```
Comment (GijsWeterings) [inline src/tindev-commits.html]: You still hardcoded this commit ;)

**Review by GijsWeterings — CHANGES_REQUESTED** [review]

File: src/tindev-commits.html (hunk 1 above)
Comment (TimvdLippe) [reply src/tindev-commits.html]: Yes, for now. Still waiting on @mpasterkamp to finish the import, such that we can hook `es-query` into it.

**Comment by TimvdLippe** [comment]
This is now ready for merging. The actual updating has to be done after @clanghout finished his model update work

**Review by clanghout — APPROVED** [review]

File: src/es-query.html [hunk 2]
```diff
@@ -58,8 +82,11 @@
         return res.slice(0, -1);
       },
 
-      _computeUrl: function(baseDomain, path, q) {
-        return baseDomain + path + 'q=' + this._stringifyQComponents(q);
+      _computeUrl: function(baseDomain, path, q, type) {
+        if (type === 'GET') {
+          return baseDomain + path + '/' + type + '?size=100&q=' + this._stringifyQComponents(q);
```
Comment (GijsWeterings) [inline src/es-query.html]: why would we include `GET` in the url?

File: src/tindev-commits.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+      }
+
+      label {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      star-rating {
+        display: inline-block;
+        --star-color: #231657;
```
Comment (GijsWeterings) [inline src/tindev-commits.html]: unused

**Review by GijsWeterings — CHANGES_REQUESTED** [review]

**Comment by TimvdLippe** [comment]
@GijsWeterings Fixed :smile:

**Review by GijsWeterings — APPROVED** [review]

**Pull request merged**